opentelemetry-otlp/CHANGELOG.md

```diff
@@ -13,6 +13,15 @@
 - Remove unnecessarily public trait `opentelemetry_otlp::metrics::MetricsClient`
   and `MetricExporter::new(..)` method. Use
   `MetricExporter::builder()...build()` to obtain `MetricExporter`.
+- The HTTP clients (reqwest, reqwest-blocking, hyper) now support the
+timeout internal configured in below order
+  - Signal specific env variable `OTEL_EXPORTER_OTLP_TRACES_TIMEOUT`, 
+  `OTEL_EXPORTER_OTLP_LOGS_TIMEOUT` or `OTEL_EXPORTER_OTLP_TIMEOUT`.
+  - `OTEL_EXPORTER_OTLP_TIMEOUT` env variable.
+  - `with_http().with_timeout()` API method of
+`LogExporterBuilder` and `SpanExporterBuilder` and `MetricsExporterBuilder`.
+  - The default interval of 30sec is used if none is configured.
```

Comment:
https://github.com/open-telemetry/opentelemetry-specification/blob/714b1ae1382ba57850f29ee94b854b2aa7d1be83/specification/protocol/exporter.md?plain=1#L67 should it be 10 instead?

Reply:
Good catch. It's indeed 10sec. Have update the changelog accordingly.

https://github.com/open-telemetry/opentelemetry-rust/blob/dde68a073c5c960708b0bfc6c7cc2a8f5f069b17/opentelemetry-otlp/src/exporter/mod.rs#L55